src/styles/calculator.css

```diff
@@ -0,0 +1,67 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
- [ ] We can also add a background-color to the calculator so make it match the design `.calculator {
  background-color: grey;
}`